### PR TITLE
Admin manage user

### PR DESCRIPTION
**Pull request by haonh**

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ def initialize(user)
     user ||= User.new # guest user (not logged in)
     if user.admin?
       can :manage, :all
+    end
     if !user.id.nil?
```
Comment (nhs3108) [inline app/models/ability.rb]: Sửa đê :))

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class Admin::UsersController < ApplicationController
+
```
Comment (ttuan) [inline app/controllers/admin/users_controller.rb]: remove this line

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
 
+  namespace :admin do
+    root "users#index"
+    resources :users
```
Comment (thiennv57) [inline config/routes.rb]: only

File: app/views/admin/users/_user.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<tr id="user-<%= user.id%>">
```
Comment (ttuan) [inline app/views/admin/users/_user.html.erb]: add a space before %>

File: app/views/admin/users/_user.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  <td><%= link_to user.name, user %></td>
+  <td><%= user.email %></td>
+  <td>
+    <%= form_for [:admin, user], remote: true do |f| %>
+      <%= f.hidden_field :admin, value: !user.admin %>
+      <%= f.submit user.admin? ? t("admin.set_normal_user") : t("admin.set_admin_user"), 
+        class: "btn btn-primary btn-sm" %>
+    <% end %>
+  </td>
+  <td><%= link_to "delete", admin_user_path(user), remote: true, method: :delete,
```
Comment (ttuan) [inline app/views/admin/users/_user.html.erb]: i18n

File: app/views/admin/users/destroy.html.erb [hunk 6]
```diff
@@ -0,0 +1 @@
+$("#user-<%= @user.id%>").remove();
```
Comment (ttuan) [inline app/views/admin/users/destroy.html.erb]: add a space before %>

File: app/models/ability.rb [hunk 7]
```diff
@@ -5,7 +5,8 @@ def initialize(user)
     user ||= User.new # guest user (not logged in)
     if user.admin?
       can :manage, :all
-    if !user.id.nil?
+    end
+    if !user.nil?
```
Comment (thiennv57) [inline app/models/ability.rb]: unless

File: app/views/admin/users/update.js.erb [hunk 8]
```diff
@@ -0,0 +1 @@
+$("#user-<%= @user.id%>").replaceWith("<%= escape_javascript(render "user", user: @user) %>");
```
Comment (ttuan) [inline app/views/admin/users/update.js.erb]: add a space

**Pull request merged**